### ISSUE TITLE
Cancel Button doesn't restore original values

### ISSUE THREAD

**Issue by elyday** [dlemmermann/PreferencesFX#68]
Hello,

I am currently trying to use PreferencesFX and I have encountered a problem or even a bug directly.

I created a simple demo application with PreferencesFX, with a StringProperty, an IntegerProperty and a BooleanProperty. I put a listener on the StringProperty, which always gives me the latest content when changes are made.

```
private final StringProperty welcomeText = new SimpleStringProperty("Hello World");
this.welcomeText.addListener((observable, oldValue, newValue) -> System.out.println(newValue));
```

This allows me to see that the changes are not undone when leaving the window via the Cancel button. Is this a bug or do I still have to adjust other parts to make the Cancel button work properly?

My entire demo application is available in the following Gist: https://gist.github.com/elyday/64a0975551fb7121b874491bc6eaef33

**Comment by elyday** [comment]
I checked out the develop branch once, because I actually intended to write for a fix here, and I noticed that this bug has already been fixed in the develop branch.

Maybe they are ready to release a new version with this fix?

**Comment by martinfrancois** [comment]
Hi,
Thanks for the detailed explanation!
The fix for this bug was already implemented in #56.
We will likely make a new release soon, in the meantime you can build the develop branch and install PreferencesFX in your local repository to use it.

**Comment by martinfrancois** [comment]
Hi @elyday 
The fix for this issue was included in release `8.3.0` (for Java 8) and `11.3.0` (for Java 11), which has just been released.
